EventListener/ViewResponseListener.php

```diff
@@ -72,9 +70,6 @@ public function onKernelView($event)
         }
 
         if ($configuration instanceof ViewAnnotation) {
```

Comment:
`$configuration = $request->attributes->get('_template');` where does the _template attribute get set? It seems the Sensio FrameworkExtraBundle can set this attribute. But how can it contain the FosRestBudnle view annotation?

Reply:
IIRC that's happening because or `@View` annotation extends the `@Template` annotation from the SensioFrameworkExtraBundle.

Reply:
I see. I think we should also remove this inheritance.